torch/onnx/_internal/fx/fx_onnx_interpreter.py

```diff
@@ -248,6 +248,11 @@ def _fill_tensor_shape_type(
                 expected_value
             )
             onnxscript_value.shape = torch.Size([1])
+        elif isinstance(expected_value, (int, float, bool)):
```

Comment:
str as well?

Why does the op return scalar?
https://github.com/pytorch/pytorch/blob/68f74dd16292cbdd2815ad7c7e9ae9cbac2785e0/aten/src/ATen/native/native_functions.yaml#L14484

Reply:
Update the description to be `aten::_scaled_dot_product_flash_attention`. I think int got returned for the SymInt output.

Reply:
I think this might be the first case where we see an op returning sym whose value is actually constant. My guess is normally for op with single output returning sym, if the sym turns out to be static, it gets traced as a scalar directly.

Reply:
> Not sure how string type translates to onnx. Do we have use case for onnx::constant tensor of string?

Make sense. I have seen placeholder with str, but you are right, this is in the middle of conversion which we don't get str from onnx output.

Reply:
With the added test, we should find out soon :)